Jamulus.pro

```diff
@@ -410,7 +410,14 @@ win32 {
         }
         ICONSDIR = $$absolute_path($$ICONSDIR, $$PREFIX)
         icons.path = $$ICONSDIR
-        icons.files = distributions/jamulus.png distributions/jamulus.svg distributions/jamulus-server.svg
+        icons.files = distributions/jamulus.png
+
+        isEmpty(ICONSDIR_SVG) {
+            ICONSDIR_SVG = share/icons/hicolor/scalable/apps/
+        }
```

Comment:
I've no idea why we are checking `ICONSDIR` (and others) for emptiness. Those variables do not seem magical (i.e. they are not pre-set by qmake, as far as I can see). Therefore the intention was probably to make them overridable. Anyway, I'm following this path and I'm using the same pattern (probably same reasoning what @atsampson did in 69e3286057a94574ba7b61e7f93ec812251775d2 when first adding it for `ICONSDIR`).

Fear of breakage is what prevents me from removing that. :)